From ec3b7496ff910daf9f7c50e3c02db35396ec27a7 Mon Sep 17 00:00:00 2001
From: Tristan Bastian <tristan-c.bastian@gmx.de>
Date: Wed, 16 Jan 2019 19:02:19 +0100
Subject: [PATCH 1/2] nyan-big: change spi delay

Internal keyboard of nyan-big is only working when cold booting by pressing [reload/refresh]+[power] button.
With this patch keyboard is working by only pressing [power] button.

Signed-off-by: Tristan Bastian <tristan-c.bastian@gmx.de>
---
 arch/arm/dts/tegra124-nyan-big-u-boot.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi b/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi
index 3ff2f67c1a..ddfeba806c 100644
--- a/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi
+++ b/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi
@@ -15,7 +15,7 @@
        };
 
        spi@7000d400 {
-               spi-deactivate-delay = <200>;
+               spi-deactivate-delay = <500>;
                spi-max-frequency = <3000000>;
 
                cros_ec: cros-ec@0 {
-- 
2.19.1
